finetuner/tuner/onnx.py

```diff
@@ -0,0 +1,207 @@
+from typing import Any, List, Tuple, Union
+
+import numpy as np
+import onnxruntime
+
+from finetuner.helper import AnyDNN
+from finetuner.helper import get_framework
+
+
+def to_onnx(
+    embed_model: AnyDNN,
+    path: str,
```

Comment:
i suggest we use `model_path` here, consistent with what we have in `def save(..)`. (see tuner/__init__.py)

Reply:
Good point. I changed it.